Specs/Renderer/BuiltinFunctionsSpec.js

```diff
@@ -316,7 +316,9 @@ describe(
     it("has czm_decompressTextureCoordinates", function () {
       var fs =
         "void main() { " +
-        "  gl_FragColor = vec4(czm_decompressTextureCoordinates(8386559.0) == vec2(0.4998779, 0.4998779)); " +
+        "  vec2 tmp = czm_decompressTextureCoordinates(8386559.0); " +
+        "  bool ok = all(lessThanEqual(abs(tmp - vec2(0.4998779, 0.4998779)), vec2(.00000005)));" +
+        "  gl_FragColor = vec4(ok); " +
```

Comment:
Thanks @markw65 - can confirm this fixes the spec failure on macOS in both Chrome and Firefox. Just one suggestion for readability and I think this is good to go -
```suggestion
        "  vec2 coords = czm_decompressTextureCoordinates(8386559.0); " +
        "  vec2 expected = vec2(0.4998779, 0.4998779);" +
        "  gl_FragColor = vec4(all(lessThanEqual(abs(coords - expected), vec2(0.00000005)))); " +
```